src/modules/ekf2/EKF/covariance.cpp

```diff
@@ -528,14 +528,19 @@ bool Ekf::checkAndFixCovarianceUpdate(const SquareMatrixState &KHP)
 void Ekf::resetQuatCov(const float yaw_noise)
 {
 	const float tilt_var = sq(fmaxf(_params.initial_tilt_err, 1.0e-2f));
-	Vector3f rot_var_ned(tilt_var, tilt_var, 0.f);
+	float yaw_var = 0.0f;
```

Comment:
```suggestion
	float yaw_var = sq(0.01f);
```

Reply:
Init to a minimum then if there's a caller provided noise take it below if larger?